status/discover.go

```diff
@@ -136,10 +136,16 @@ func checkTracingService() (url string) {
 	return tracingConfig.URL
 }
 
-func DiscoverJaeger() string {
+// DiscoverJaeger will return the Jaeger URL if it has been configured.
+// AuthToken is the users auth token. When is not empty, it'll check if
+// user has access to Jaeger service.
+func DiscoverJaeger(authToken string, auth *config.Auth) string {
+	url := ""
+
```

Comment:
The return path below is not covered by the authentication check. Since the state-defined URL can be accessed with different user tokens, it should be checked as well.